Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/service/impl/ProcessServiceImpl.java

```diff
@@ -354,9 +354,9 @@ public ExportFormatResultType exportProcess(final String name, final Integer pro
             org.springframework.security.core.Authentication auth =
                 org.springframework.security.core.context.SecurityContextHolder.getContext().getAuthentication();
             if (auth != null) {
-                LOGGER.info("Authentication principal=" + auth.getPrincipal() + " details=" + auth.getDetails() + " thread=" + Thread.currentThread());
+                LOGGER.debug("Authentication principal={} details={} thread={}", auth.getPrincipal(), auth.getDetails(), Thread.currentThread());
```

Comment:
This must be LOGGER.info, not debug level.

Reply:
This is explicitly marked with a comment as "Debug tracing of the authenticated principal" which is pretty solid evidence it was always intended to be debugging.  I propose to remove it rather than make it INFO level.  Agreeable?

Reply:
Removed.